lib/screen/home/home_page.dart

```diff
@@ -165,6 +165,15 @@ class _HomePageState extends State<HomePage>
     super.didPushNext();
   }
 
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) async {
+    if (state == AppLifecycleState.paused) {
+      await MetricClient.sendMetrics();
```

Comment:
can we handle the device's foreground/background state using `flutter_fgbg`?
It's already set up, and can be added it in the `_handleBackground` function.